app/main.py

```diff
@@ -348,7 +109,7 @@ async def _calculate_per_user_costs(
 
 @command_tree.command(name=command_name("monthlycost"))
```

Comment:
All of these other commands are still TODO for future PRs. Just focusing on core messaging in this PR.